market/market/coingecko/coingecko_test.go

```diff
@@ -102,3 +104,58 @@ func Test_normalizeTickers(t *testing.T) {
 		})
 	}
 }
+
+func Test_normalizeTickersWithFilter(t *testing.T) {
+	prices := coingecko.CoinPrices{
+		{
+			Id:           "cUSDC",
+			Symbol:       "cUSDC",
+			CurrentPrice: 0.0021,
+			MarketCap:    minimalMarketCap + 1,
+		},
```

Comment:
why do you need this?

Reply:
we need coin and price object for normalize test